cmd/cli/dashboard.go

```diff
@@ -108,67 +106,43 @@ func (d *dashboardCmd) run() error {
 	// Select pod/s given the service data available
 	set := labels.Set(svc.Spec.Selector)
 	listOptions := metav1.ListOptions{LabelSelector: set.AsSelector().String()}
-	pods, err := v1ClientSet.Pods(settings.Namespace()).
-		List(context.TODO(), listOptions)
+	pods, _ := v1ClientSet.Pods(settings.Namespace()).List(context.TODO(), listOptions)
```

Comment:
Shouldn't we be handling the error here? This looks like it would be fine in the case where the list comes back empty, but if some other error occurs where `pods` comes back `nil`, then dereferencing `pods.Items` will panic.

Reply:
Good catch.